usr/src/uts/common/fs/zfs/zfs_vfsops.c

```diff
@@ -2033,10 +2033,9 @@ zfs_resume_fs(zfsvfs_t *zfsvfs, const char *osname)
 	 * objset_t, as the one we had before may have been evicted.
```

Comment:
Oh, my apologies.  I meant to make this fix, but then it slipped my mind.
I see that you have sent an RTI, is it pointless for me to update this PR now?
Or should I still do it?
Thank you!